internal/sdkv2provider/schema_cloudflare_zone_settings_override.go

```diff
@@ -202,34 +202,6 @@ var resourceCloudflareZoneSettingsSchema = map[string]*schema.Schema{
 		},
 	},
 
-	"mobile_redirect": {
```

Comment:
if you're removing the schema, we'll need a schema migration here too in order to remove it from user state files. you can see some prior art in #1211

Reply:
I've tried to do something like the PR you mentioned. The whole change v0 -> v1 change is just copy-pasting the value and removing the `mobile_redirect` setting from both `settings` and `initial_settings`. Let me know if there's anything else I should change.

Reply:
yep! that's all that is needed here, thanks!